primitives/beefy/src/mmr.rs

```diff
@@ -41,8 +54,8 @@ pub struct MmrLeaf<BlockNumber, Hash, MerkleRoot> {
 	pub parent_number_and_hash: (BlockNumber, Hash),
 	/// A merkle root of the next BEEFY authority set.
 	pub beefy_next_authority_set: BeefyNextAuthoritySet<MerkleRoot>,
-	/// A merkle root of all registered parachain heads.
-	pub parachain_heads: MerkleRoot,
+	/// A merkle root hash of an arbitrary extra leaf data.
```

Comment:
technically not a merkle hash
```suggestion
	/// Arbitrary extra leaf data to be used by downstream pallets to include custom data in the [`MmrLeaf`]
```